utils/data-wranglers.ts

```diff
@@ -288,13 +288,36 @@ export const getSlimPokemonData = ({
   key?: "id" | "slug";
   id: string;
 }) => {
-  const { id: pid, slug, flavorText, types, colorPalette } = getPokemonDetails({
+  const {
+    id: pid,
+    slug,
+    flavorText,
+    types,
+    colorPalette,
+  } = getPokemonDetails({
     key,
     id,
   });
   return { id: pid, slug, flavorText, types, colorPalette };
 };
 
+export const getSlimPokemonDataGraphQL = (response: Record<string, any>) => {
+  const pid = response.id;
+  const colorPalette = pokemonColorPalettes["1"];
```

Comment:
This is going to be the real kicker, I wonder if we can get the color and generate the pallet. Currently the API gives us something like "red" and "green"